mesh/pmesh.cpp

```diff
@@ -1292,6 +1310,24 @@ void ParMesh::Finalize(bool refine, bool fix_orientation)
    FinalizeParTopo();
 }
 
+int ParMesh::GetLocalElementNum(long global_element_num) const
+{
+   if (!have_global_element_offset) {
+      ComputeGlobalElementOffset();
+   }
+   int local = global_element_num -global_element_offset;
```

Comment:
Missing space after '-' (just to be consistent with most of the code)

Reply:
Agree.  Whole thing needs a make style I think.